mkdocs.yml

```diff
@@ -27,6 +27,13 @@ nav:
     - Build system: build-system.md
     - Communicate with JavaScript: communicate-with-javascript.md
     - Melange for X developers: melange-for-x-developers.md
+    - Melange Book:
```

Comment:
I believe that mkdocs don't like much having 2 levels nesting in the menu, because when I click on some other section, the "Melange Book" one stays opened:

![mkdocs-weird](https://github.com/melange-re/melange-re.github.io/assets/220424/9508023d-4a86-4984-ba22-a79faab43b8a)

Not a big deal for now, but just something to consider if we end up leaving the book somewhere around here.